src/routing/approved.rs

```diff
@@ -99,6 +101,7 @@ impl Approved {
             network: Network::new(),
             section_keys_provider,
             vote_accumulator: Default::default(),
+            message_accumulator: Default::default(),
```

Comment:
Suggestion for a possible future refactoring (not necessary to be addressed in this PR): instead of adding another accumulator for this, we could have reused `vote_accumulator` instead, and just created a new `Vote` variant for the dst-accumulated messages. That way we would avoid duplicating the accumulation logic.

Reply:
the `vote_accumulator` is just a wrapper of `SignatureAggregator` , which I think shall be OK to be used directly ?
maybe we shall re-use it in this PR ?

Reply:
> it may little bit consusing with the existing SendMessage entry

True, but I think it's just a matter of naming. So we would have two vote variants both dealing with message accumulation. One is for accumulating section-src messages and one for node-src messages. So if we reflect this distinction in the name of the respective vote variants, then there shouldn't be much confusion.